microsoft-edge/extensions-chromium/publish/api/addons-publish-api-reference.md

```diff
@@ -0,0 +1,373 @@
+---
+description: The Add-ons Publish API Reference, for REST endpoints to automate publishing updates to add-ons that are submitted to the Microsoft Edge Add-ons website.
+title: Add-ons Publish API Reference
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 08/10/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, publish api, add-ons publish api
+---
+# Add-ons Publish API Reference
```

Comment:
Possible title of article:
- API Reference for publishing add-ons for Microsoft Edge
- Publish Add-ons API Reference

Reply:
API Reference for publishing add-ons for Microsoft Edge

This seems better. What will be the title on the left navigation?

Reply:
See the very bottom of the Files tab of this repo, I put a comment for the TOC there.  
Current short title in TOC: 
**Add-ons Publish API Reference**
Suggested revised short title in TOC:
**Publish Add-ons API Reference**